### PR TITLE
Code review of CG work

### PR DESCRIPTION
**Pull request by wkozmaNTIA**

### CODE REVIEW

File: src/p528/LineOfSight.cpp
Comment (blainNTIA) [inline src/p528/LineOfSight.cpp:165]: The var name `result` is already used (a param). Might be good to rename this one.

File: src/p528/RayOptics.cpp
Comment (blainNTIA) [inline src/p528/RayOptics.cpp:29]: `path` is not used. Can it be removed?

File: src/p528/SmoothEarthDiffraction.cpp
Comment (blainNTIA) [inline src/p528/SmoothEarthDiffraction.cpp:97]: This is never used. Can it be removed?

File: win32/Manual/Program.cs
Comment (blainNTIA) [inline win32/Manual/Program.cs:143]: Unreachable code

**Review by blainNTIA — APPROVED** [review]
Great code! I made a number of comments to call your attention to possible minor issues. I am approving with the assumption you'll look at my comments and make any changes you feel are needed. Also, it looks like the P.528 changed to rev 5 but there are still a few rev 4 references.
```
$ git grep -n "P.528-4"
README.md:1:# Recommendation ITU-R P.528-4 - U.S. Reference Implementation  #
src/p528/AbsorptionParameters.cpp:15: |                Recommendation ITU-R P.528-4, "Propagation curves for
src/p528/CalculateEffectiveRayLength.cpp:14: |                in Annex 2, Section 12 of Recommendation ITU-R P.528-4,
win32/Manual/Program.cs:209:        /// Perform a diff with the P.528-4 data table files
```

File: include/p676.h
Comment (wkozmaNTIA) [reply include/p676.h]: added comments to first struct; removed from next two

File: src/p528/GetPathLoss.cpp
Comment (wkozmaNTIA) [reply src/p528/GetPathLoss.cpp:38]: good catch.  must have been removed at some point in the past during a refactor but never updated the function signature

File: src/p676/RayTrace.cpp
Comment (wkozmaNTIA) [reply src/p676/RayTrace.cpp:29]: yep.  added.

File: src/p835/MeanAnnualGlobalReferenceAtmosphere.cpp
Comment (wkozmaNTIA) [reply src/p835/MeanAnnualGlobalReferenceAtmosphere.cpp:130]: actually, that is correct for the units.  geometric height is km while geopotential height is km'

File: win32/Manual/Program.cs
Comment (wkozmaNTIA) [reply win32/Manual/Program.cs:100]: manual test code that will be removed before merge

File: win32/Manual/Program.cs
Comment (wkozmaNTIA) [reply win32/Manual/Program.cs:130]: manual test code that will be removed before merge

File: win32/Manual/Program.cs
Comment (wkozmaNTIA) [reply win32/Manual/Program.cs:143]: manual test code that will be removed before merge

File: src/p528/LineOfSight.cpp
Comment (wkozmaNTIA) [reply src/p528/LineOfSight.cpp:90]: yeah, you're right.  and it can fit on the first line

File: src/p528/LongTermVariability.cpp
Comment (wkozmaNTIA) [reply src/p528/LongTermVariability.cpp:20]: agreed.  should be consistent like that

File: src/p528/NakagamiRice.cpp
Comment (wkozmaNTIA) [reply src/p528/NakagamiRice.cpp:29]: yes, as the range of K starts at a minimum value of -40 in the vector

File: src/p676/GaseousAttenuation.cpp
Comment (wkozmaNTIA) [reply src/p676/GaseousAttenuation.cpp:9]: hum, actually this code is dead and never called.  just going to delete it.

File: src/p676/RefractiveIndex.cpp
Comment (wkozmaNTIA) [reply src/p676/RefractiveIndex.cpp:2]: added

File: P528Drvr/P528Drvr.cpp
Comment (wkozmaNTIA) [reply P528Drvr/P528Drvr.cpp:419]: i believe you are correct and the `if (size)` conditional check can be removed

File: src/p528/RayOptics.cpp
Comment (wkozmaNTIA) [reply src/p528/RayOptics.cpp:29]: yep.  correct.

File: src/p528/SmoothEarthDiffraction.cpp
Comment (wkozmaNTIA) [reply src/p528/SmoothEarthDiffraction.cpp]: actually i should replace the const `1.607` in the following lines with `B_0` to add clarity

File: src/p835/MeanAnnualGlobalReferenceAtmosphere.cpp
Comment (blainNTIA) [reply src/p835/MeanAnnualGlobalReferenceAtmosphere.cpp:130]: Interesting. I did not know that.

File: src/p528/TranshorizonSearch.cpp
Comment (wkozmaNTIA) [reply src/p528/TranshorizonSearch.cpp]: yep

File: src/p676/SlantPath.cpp
Comment (wkozmaNTIA) [reply src/p676/SlantPath.cpp:15]: removed.  h_1 can't be on the surface of the Earth since it has a min value of 1.5 meters.

File: src/p676/SlantPath.cpp
Comment (wkozmaNTIA) [reply src/p676/SlantPath.cpp:45]: correct.  the loop will always terminate now so no need for an error condition check

**Pull request merged**
